### DISCUSSION TITLE
Do react-pdf support pdf form fields (TextInput)?

### DISCUSSION THREAD

**Discussion by lthripurani** [diegomura/react-pdf#1661] [Q&A]
Do react-pdf support pdf form fields (TextInput)?

**Comment by jeetiss** [comment]
no

**Reply by lthripurani** [reply]
thank you

**Reply by lthripurani** [reply]
Is there any plan to include this component in future?

**Reply by lthripurani** [reply]
Please help in replying

**Reply by asherccohen** [reply]
I really like the work done here, I'm also interested in knowing if there are plans (or workarounds) to make editable fields available?
Would be a killer feature!
